### PR TITLE
[FLINK-18860][docs-zh] Translate "Execution Plans" page of "Managing Execution" into Chinese

### PR DESCRIPTION
**Pull request by Shawn-Hx**
## What is the purpose of the change

Translate "Execution Plans" page of "Managing Execution" into Chinese.

The page url is https://ci.apache.org/projects/flink/flink-docs-master/zh/dev/execution_plans.html

The markdown file is located in `flink/docs/dev/execution_plans.zh.md`

## Brief change log

Translate `flink/docs/dev/execution_plans.zh.md`

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable)

### CODE REVIEW

**Comment by Shawn-Hx** [comment]
Hi, @wuchong .
This PR is ready to be reviewed. Would you please help to review it at your convenience?
Thank you!

**Comment by Shawn-Hx** [comment]
Hi, @klion26 .
This PR is ready to be reviewed. Would you please help to review it at your convenience ?
Thank you ~

File: docs/dev/execution_plans.zh.md [hunk 1]
```diff
@@ -54,24 +51,20 @@ println(env.getExecutionPlan())
 </div>
 </div>
 
+按照下列步骤来可视化执行计划：
```
Comment (klion26) [inline docs/dev/execution_plans.zh.md:54]: `可以通过如下步骤可视化执行计划` 会好一些吗？

**Review by klion26 — COMMENTED** [review]
@Shawn-Hx thanks for the contribution, left one minor comment.

File: docs/dev/execution_plans.zh.md (hunk 1 above)
Comment (Shawn-Hx) [reply docs/dev/execution_plans.zh.md]: Thanks for your advice! I have changed this sentence, it looks better now.

**Review by klion26 — APPROVED** [review]
@Shawn-Hx thanks for addressing my comments. LGTM now
ping @wuchong @rmetzger

**Comment by Shawn-Hx** [comment]
kindly ping @wuchong @rmetzger

**Comment by wuchong** [comment]
Merging...

**Pull request merged**